docker/hisat2/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:17.04
+MAINTAINER Jishu Xu <jishuxu@broadinstitute.org>
```

Comment:
Maintainer is deprecated; recommended style is now to add a label with maintainer, and to link labels together in one line to reduce number of layers in the image. See the images in master for examples.

Reply:
1. Vote for use labels, would look like below:
```
LABEL maintainer="Jishu Xu <jishuxu@broadinstitute.org>" \
  software="HISAT2" \
  version="2-2.1.0" \
  description="RNA-seq aligner" \
  website="https://ccb.jhu.edu/software/hisat2/index.shtml"
```
2. I noticed that sometimes we use `FROM ubuntu:17.04`, sometimes we use `FROM ubuntu:zesty`, although they are the same thing, I'd rather we could use a unified name here in Dockerfile, maybe a template Dockerfile could address this?

Reply:
changed in dockerfile.